performance_regression_test.py

```diff
@@ -672,3 +677,136 @@ def test_timeseries_bench(self):
+        """
+        Test steps:
+
+        1. Run a write workload as a preparation
+        2. Run a mixed workload
+        """
+        stress_cmd_r_templ = self.params.get('stress_cmd_r')
+        start_ops = throttle_step = 10_000
+        max_ops = 150_000
+        stress_num = 1
+        num_loaders = len(self.loaders.nodes)
+
+        self.run_fstrim_on_all_db_nodes()
+        # run a write workload as a preparation
+        compaction_strategy = self.params.get('compaction_strategy')
+        self.preload_data(compaction_strategy=compaction_strategy)
+        self.wait_no_compactions_running(n=400, sleep_time=120)
+        # run a mixed workload
+        # create new document in ES with doc_id = test_id + timestamp
+        # allow to correctly save results for future compare
+        # self.create_test_stats(doc_id_with_timestamp=True)
+        self.run_fstrim_on_all_db_nodes()
+        # self.disable_autocompaction_on_all_nodes()
+        self.run_increasing_by_step_ops_load(stress_cmd_r_templ,
+                                             start_ops, max_ops, throttle_step,
+                                             stress_num, num_loaders, compaction_strategy)
+
+    # pylint: disable=too-many-arguments,too-many-locals
+    def run_increasing_by_step_ops_load(self, stress_cmd_templ,
+                                        start_ops, max_ops, throttle_step,
+                                        stress_num, num_loaders, compaction_strategy):
+        total_summary = []
+        for current_ops in range(start_ops, max_ops + throttle_step, throttle_step):
+            # calculate throttle: current_ops / ( 4 loaders * stress_num)
+            #        10000 / (4 * 1)  = 2500
+            # num_threads = 50 * (current_ops // 1000 // 51 + 1) // stress_num
+            num_threads = 700
+            current_throttle = current_ops // (num_loaders * stress_num)
+            stress_cmd_m_list = list(map(lambda x: x.replace("$threads", f"{num_threads}").replace("$throttle", f"{current_throttle}"),  # pylint: disable=cell-var-from-loop
+                                         stress_cmd_templ))
+            stress_queue = self.run_stress_thread(stress_cmd=stress_cmd_m_list[0], stress_num=stress_num,
+                                                  compaction_strategy=compaction_strategy, stats_aggregate_cmds=False)
+            results = self.get_stress_results(queue=stress_queue, store_results=False)
+            summary_result = self._calculate_average_latency(results)
+            summary_result["ops"] = current_ops
+            self.log.info("C-S results for ops: %s. \n Results: \n %s", current_ops, summary_result)
+            self.wait_no_compactions_running(n=400, sleep_time=120)
+            total_summary.append(summary_result)
+            self.log.info("Final table with results: \n %s", pprint.pformat(total_summary))
+
+    def disable_autocompaction_on_all_nodes(self, ks="keyspace1", cf="standard1"):
```

Comment:
i'm not sure why you are disabling autocompaction here, and not sure if you should do it here, but if you do, this function was already implemented [here](https://github.com/scylladb/scylla-cluster-tests/blob/2c05a2270c6f7375a3d32b8487e410a1f1a0feee/stop_compaction_test.py#L41)